src/modules/commerce/cart/index.ts

```diff
@@ -5,12 +5,12 @@ import { CartData, CartItemData, CartItemInput } from '../../../types/commerce';
 export class Cart {
   constructor(protected client: ClientType) { }
 
-  public async addToCart(input: CartItemInput): Promise<CartItemData> {
+  public async addToCart(input: CartItemInput[]): Promise<CartItemData> {
```

Comment:
must be a mayor like: 0.16.0 because this is a breaking change

Reply:
@buttercubz done